lms/djangoapps/grades/course_grades.py

```diff
@@ -14,7 +14,7 @@
 log = getLogger(__name__)
 
 
-GradeResult = namedtuple('StudentGrade', ['student', 'gradeset', 'err_msg'])
+GradeResult = namedtuple('GradeResult', ['student', 'gradeset', 'err_msg'])
```

Comment:
Python doesn't have enough information at runtime to be able to care about this.  You can assign any object to any name you want.  I got careless, and changed the variable, without changing the name of the tuple.